src/MultiTypeSymbolTable.chpl

```diff
@@ -195,16 +194,20 @@ module MultiTypeSymbolTable
 
         :arg name: name of the array
         :type name: string
+
+        :returns: bool indicating whether deletion was successful
```

Comment:
As above, I recommend changing the wording to "bool indicating whether the deletion occurred"